fourth-challenge/resources/views/cities/edit-form.blade.php

```diff
@@ -0,0 +1,47 @@
+<x-layout>
+
+<div class="space-y-10 divide-y divide-gray-900/10 py-10 px-12">
+
+  <div class="grid grid-cols-1 gap-y-8 gap-x-8 pt-10 sm:grid-cols-3">
+    <div class="px-4 sm:px-0">
+      <h2 class="text-base font-semibold leading-7 text-gray-900">Edit a city</h2>
+      <p class="mt-1 text-sm leading-6 text-gray-600">This information will be publicly display.</p>
+    </div>
+
+    <form class="edit-form bg-white shadow-sm ring-1 ring-gray-900/5 sm:rounded-xl md:col-span-1" id="{{$city->id}}">
+     @csrf
+      <div class="px-4 py-6 sm:p-8">
+        <div class="grid max-w-sm grid-cols-1 gap-y-8 gap-x-6 sm:grid-cols-1">
+{{--            TODO pasarlo a un componente--}}
```

Comment:
Same thing here with the TODOs and please choose a language and stick with it, either everything in Spanish or everything in English. One or the other in this context is the same to me but consistency is important